config_examples/arm/CentOS_7.1.json

```diff
@@ -29,6 +29,7 @@
             "ssh_username": "{{user `ssh_user`}}",
             "ssh_password": "{{user `ssh_pass`}}",
 
+            "os_type": "linux",
```

Comment:
Yes.  The os_type select the necessary workflow executed by Packer.  Linux and Windows are different.

This is based on the original ASM code...